src/App.js

```diff
@@ -23,11 +23,17 @@ class App extends Component {
     return (
       <div>
         <div>
-          <h2>Welcome to React</h2>
+          <h2>Bag It Up!</h2>
         </div>
           <div>
             { !currentUser && <SignIn/> }
-            { currentUser && <CurrentUser user={currentUser} /> }
+            { 
+              currentUser && 
+              <div>
```

Comment:
Adding new component for logged in users to view. Authorized users can now create a `<NewShoppingList/>`